src/cargo/ops/fix.rs

```diff
@@ -99,6 +104,14 @@ pub fn fix(ws: &Workspace<'_>, opts: &mut FixOptions<'_>) -> CargoResult<()> {
         wrapper.env(IDIOMS_ENV, "1");
     }
 
+    if opts.use_clippy {
+        wrapper.env(CLIPPY_FIX_ENV, "1");
+    }
+
+    if let Some(clippy_args) = &opts.clippy_args {
+        wrapper.env(CLIPPY_FIX_ARGS, clippy_args.join(CLIPPY_PASSHTHROUGH_SEP));
```

Comment:
this is the only way I could think of to consistently handle arg passthrough without brittle splitting on spaces, I'm not sure i prefer it over the more brittle option though because anything beyond 1 arg is much more verbose and annoying to use with this impl style.